README.md

```diff
@@ -872,6 +872,7 @@ _Related: [Groupware](#groupware)_
 - [instant.io](https://github.com/webtorrent/instant.io) - Streaming file transfer over WebTorrent. ([Demo](https://instant.io)) `MIT` `Nodejs`
 - [Magnetico](https://github.com/boramalper/magnetico) - Magnetico is the first autonomous (self-hosted) BitTorrent DHT search engine suite that is designed for end-users. `AGPL-3.0` `Python`
 - [Magnetissimo](https://github.com/sergiotapia/magnetissimo) - Search engine that indexes all popular torrent sites. `MIT` `Elixir`
+- [mork](https://github.com/dojje/mork) - Simple command line based file sending utility, written in rust. `GPL-3.0` `Rust`
```

Comment:
> the program will contact the tracker server which keeps track of all the codes

I understand this repository contains source code for both the command-line client and the server? How do I run the server?

Reply:
ping @AlixPlayzz